docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,500 @@
+    - `Admin.TLS.RootCAs:`  - The path to and file name of the TLS CA Root cert `tls-ca-cert.pem`.  
+    - `Admin.TLS.ClientRootCAs:` - The path to and file name of the TLS CA Root cert `tls-ca-cert.pem`.  
+    - `ChannelParticipation:Enabled` - Set this value to `true` to enable this feature on the orderer.
+- Put the [system channel into maintenance mode](../kafka_raft_migration.html) using the same process for the Kafka to Raft migration.  
+- Remove the system channel from the set of orderers, one by one. If an application channel can tolerate one server offline, you should still be able to submit transactions to the channel, via the other orderers that are not undergoing the removal at that time.  Use the `osnadmin channel list` command to view the channels that this orderer is a member of:
+    ```
+    osnadmin channel list –-orderer [ORDERER_ADMIN_LISTENADDRESS] --ca-file [TLS_CA_ROOT_CERT] --client-cert [OSN_TLS_SIGN_CERT] --client-key [OSN_TLS_PRIVATE_KEY]
+    ```
+
+    where:
+
+    - `ORDERER_ADMIN_LISTENADDRESS` corresponds to the `Orderer.Admin.ListenAddress` defined in the `orderer.yaml` for this orderer.
+    - `TLS_CA_ROOT_CERT` is the fully qualified path and file name of the orderer organization TLS CA root certificate and intermediate certificate if using an intermediate TLS CA.
+    - `OSN_TLS_SIGN_CERT` is the orderer signed certificate from the TLS CA.
+    - `OSN_TLS_PRIVATE_KEY` is the orderer private key from the TLS CA.
+
+    **Note:** The connection between the `osnadmin` CLI and the orderer requires mutual TLS, although it is not required for the network itself. This means you need to pass the `--client-cert` and `--client-key` parameters on each `osdadmin` command. The `--client-cert` parameter points to the orderer certificate issued by the TLS CA, and `--client-key` refers to the orderer private key issued by the TLS CA.  
+
+    For example:
+    ```
+    osnadmin channel list –-orderer HOST1:7081 --ca-file organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/cacerts/tls-ca-cert.pem --client-cert organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/signcerts/cert.pem --client-key organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/keystore/tls-key.pem.pem
+    ```
+
+    The output of this command looks similar to:
+
+    ```
+    {
+      "systemChannel": {
+          "name": "systemchannel",
+          "url": "/participation/v1/channels/systemchannel"
+      },
+      "channels":[
+          {
+            "name": "channel1",
+            "url": "/participation/v1/channels/channel1"
+          }
+      ]
+    }
+    ```
+    Run `osnadmin system-channel remove` to remove the system channel from the node configuration:
```

Comment:
As noted above, this is not a separate command but simply "osnadmin channel remove" and specifying the --channel-id of the system channel.